README.md

```diff
@@ -23,158 +23,19 @@ PHP-based as default, and more. It's used in [Yii Framework](https://www.yiifram
 
 ## Installation
 
-The package could be installed via composer:
+The package could be installed with [Composer](https://getcomposer.org):
 
 ```shell
 composer require yiisoft/view
 ```
 
-## General usage
```

Comment:
Readme should contain section "General usage" with example of base usage.

Reply:
English guide removed. I kept the link in Portuguese because it had already been translated.

Reply:
Can you keep README.md for english with links:

```
The package provides two use cases for managing view templates:

- [Basic functionality](docs/guide/en/basic-functionality.md) for use in any environment.
- [Advanced functionality](docs/guide/en/use-in-web-environment.md) for use in a web environment.
```

Reply:
and what the call would be like in '/readme.md'?
```
## General usage

The package provides two use cases for managing view templates:

- [Basic functionality](docs/guide/en/basic-functionality.md) for use in any environment.
- [Advanced functionality](docs/guide/en/use-in-web-environment.md) for use in a web environment.

### State of `View` and `WebView` services
```

Reply:
Add link as guide in "Documentation" section.